### PR TITLE
Whitelist keys sent to support

### PR DESCRIPTION
**Pull request by flavius**

### CODE REVIEW

**Comment by rinnhofer** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- Model/Adminhtml/Support.php  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/wirecard/magento2-ee/pullRequest?prid=4955833&bid=16378305)

File: Model/Adminhtml/Support.php [hunk 1]
```diff
@@ -233,7 +275,18 @@ public function sendrequest($postObject)
             ];
 
             if (preg_match('/^wirecard_elasticengine/i', $paymentCode)) {
-                $method['config'] = $this->scopeConfig->getValue('payment/' . $paymentCode, $scope);
+                $unsafeConfig = $this->scopeConfig->getValue('payment/' . $paymentCode, $scope);
+                $safeConfig = [];
```
Comment (tomazpu) [inline Model/Adminhtml/Support.php]: Maybe filtered is better? as there is nothing safe or unsafe or?
To make a unitest i would move the code to ma method so i can test it with an input and check the output.

**Comment by rinnhofer** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/wirecard/magento2-ee/pullRequest?prid=4955833&bid=16378305)

**Comment by rinnhofer** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/wirecard/magento2-ee/pullRequest?prid=4955833&bid=16378305)

**Pull request closed without merging**